Describe multiple modes of Compass installation (#1505)
kyma-incubator/compass: docs/compass/08-01-basic-flow-tutorial.md

2. In the **Overview** tab in your Namespace, select the **Deploy new resource** button and use the [`order-service`](./assets/order-service.yaml) file to connect the Application.

3. In the **API Rules** tab, copy the URL to your Application. You will use it in the Compass UI in the next steps.

### Register your Application in the Compass UI

1. Open a separate tab in your browser and go to `https://compass.{CLUSTER_DOMAIN}`. It will navigate you to the Compass UI. From the drop-down list on the top navigation panel, select the tenant you want to work on. For the purpose of this tutorial, select the `default` tenant. In the **Runtimes** tab, there is already the default `kymaruntime` that you can work on to complete this tutorial. Make sure that your Runtime is assigned to the `DEFAULT` scenario.

> 2. In the left navigation panel, navigate to the **Application** tab and click **Create application...** to register your Application in Compass. Choose **From scratch** from the drop-down list. For the purpose of this tutorial, name your Application `test-app`. By default, your Application is assigned to the `DEFAULT` scenario. Click **Create** button.

Reviewer: ```suggestion
2. In the left navigation panel, navigate to the **Application** tab and click **Create application...** to register your Application in Compass. Choose **From scratch** from the drop-down list. For the purpose of this tutorial, name your Application `test-app`. By default, your Application is assigned to the `DEFAULT` scenario. Click the **Create** button.
```

### Use your Application in the Kyma Console UI

1. Go back to the Kyma Console UI. You can see that the `test-app` Application is registered in the **Applications/Systems** view. Select `test-app` and bind it to your Namespace by selecting the **Create Binding** button.

> 2. From the drop-down list in the top-right corner, select your Namespace and go to the **Catalog** view. You will see your services available under the **Services** tab. Provision the service instance by choosing your Package as selected plan and clicking the **Add** button in the top-right corner of the page.

Reviewer: ```suggestion
2. From the drop-down list in the top-right corner, select your Namespace and go to the **Catalog** view. You will see your services available under the **Services** tab. Provision the service instance by choosing your Package as selected plan and clicking the **Add** button in the top-right corner of the page.
```
`as selected plan` -> what do you mean?

Author: There is a dropdown called "Selected plan". From here you have to select your Package.

Reviewer: ```suggestion
2. From the drop-down list in the top-right corner, select your Namespace and go to the **Catalog** view. You will see your services available under the **Services** tab. Provision the service instance by choosing your Package as **Selected plan** from the dropdown list and clicking the **Add** button in the top-right corner of the page.
```
Or you can skip the `from the dropdown list` if you think it's too lengthy.

3. Create a lambda function. In the **Overview** tab, click the **Deploy new resource** button and upload the file with the [lambda function](./assets/lambda.yaml).

4. Bind your lambda:

    a. In the left navigation panel, go to the **Lambdas** tab and click the `call-order-service` lambda.

>     b. Scroll down to the end of the Configuration view and bind your lambda to your instance by clicking the **Create Service Binding** button in the **Service Binding** section. Choose the ServiceInstance you want to bind your lambda to and click **Create Service Binding**.

Reviewer: Are they still lambdas or are they Functions now? 
```suggestion
    b. Scroll down to the end of the Configuration view and bind your function to your instance by clicking the **Create Service Binding** button in the **Service Binding** section. Choose the ServiceInstance you want to bind your function to and click **Create Service Binding**.
```

>     c. Save the settings in the right top-right corner of the page.

Reviewer: ```suggestion
    c. Save the settings in the top right corner of the page.
```

>     d. Click the **Lambdas** tab and wait until the lambda status is completed and marked as `1/1`.

Reviewer: Are they still lambdas or are they Functions now?
```suggestion
    d. Click the **Functions** tab and wait until the function status is completed and marked as `1/1`.
```

Author: Right, they're Functions now.

> 5. Expose your lambda: 

Reviewer: Are they still lambdas or are they Functions now?
```suggestion
5. Expose your function: 
```

    a. In the left navigation panel, go to the **API Rules** tab and click the **Add API Rule** button.

>     b. Type `test-lambda` in both Name and Hostname fields. From the **Service** dropdown, select `call-order-service`. Keep the **Access strategies** as they are.

Reviewer: Are they still lambdas or are they Functions now?
```suggestion
    b. Type `test-function` in both the **Name** and **Hostname** fields. From the **Service** dropdown, select `call-order-service`. Keep **Access strategies** as they are.
```

>     c. Click **Create** button at the top right corner of the page.

Reviewer: ```suggestion
    c. Click the **Create** button in the top right corner of the page.
```

> 6. Test your lambda. Open terminal app and run the following command:

Reviewer: Are they still lambdas or are they Functions now?
```suggestion
6. Test your function. Open the terminal and run the following command:
```

    ```bash

>     curl https://test-lambda.{clusterDomain}

Reviewer: ```suggestion
    curl https://test-lambda.{CLUSTER_DOMAIN}
```

>     curl https://test-lambda.{CLUSTER_DOMAIN}

Reviewer: ```suggestion
    curl https://test-function.{CLUSTER_DOMAIN}
```
I guess this one should be `test-function`, too, right?

    ```

>     You can see the following output:

Reviewer: ```suggestion
    This returns the following output:
```

    }
    ```

>     You can test your lambda by performing the following actions:

Reviewer: Are they still lambdas or are they Functions now?
```suggestion
    You can test your function by performing the following actions:
```

    - `{"action":"add"}` - adds the new order
    - `{"action":"list"}` - lists all orders (this is the default command executed after you click the **Send** button)
    - `{"action":"delete"}` - deletes all the orders

    To perform an action, run the following command:

    ```bash

>     curl --header "Content-Type: application/json" --request POST --data '{selectedAction}' https://test-lambda.{clusterDomain}

Reviewer: Are they still lambdas or are they Functions now?
```suggestion
    curl --header "Content-Type: application/json" --request POST --data '{SELECTED_ACTION}' https://test-function.{CLUSTER_DOMAIN}
```
If they're still lambdas, please edit my suggestion to apply the rest of the changes.
